spec/workers/purge_sensitive_data_worker_spec.rb

```diff
@@ -28,16 +28,21 @@
     let(:submission) do
  create(:submission, :with_completed_use_case_one_hmrc_interface_result, first_name: 'Rosie', last_name: 'Conway', 
 nino: 'JC654321A', dob: '1977-03-08'.to_date, bulk_submission:) end
-    let(:expires_at) { 1.day.ago.midnight }
+    let(:expires_at) { nil }
 
     before do
+      scheduled_time = Time.zone.local(2023, 0o6, 14, 20, 0, 0)
```

Comment:
i can see it works but what is the `0o6`. it seems to do the same thing with Time.zone.local(2023, 6, 14, 20, 0, 0).

**nit** Alternative which may be easier to read but either way
```
Time.current.midnight + 20.hours
```

Reply:
actually don't you need to fix the time to current date, rather than a fixed date?